### ISSUE TITLE
Add DESY port of TMCDB to MySQL to ACS

### ISSUE THREAD

**Issue by moraklein** [ACS-Community/ACS#28]
The TMCDB as developed for ALMA is supported on two databases, hsqldb, a lightweight java RDBMS suitable for unit testing or small installations, and Oracle, an expensive commercial DB chosen for ALMA production use. MySQL is an open-source alternative to Oracle, and in view of the unsuitability of Oracle (or probably any standard RDBMS) for the storage and retrieval of large time-series (monitor) data, probably sufficient for most configuration uses.

The implementation has already been done by Khushbu Verma of DESY, and would need only to be reviewed and tested by the community, before integration as an alternative backend.

**Comment by khushbuv84** [comment]
Hello Matias, 

Thanks for adding this issue!
However, I will upload the code on github once it is approved by Alessandro as we have already agreed upon it. 

Thanks & Best regards

**Comment by khushbuv84** [comment]
Hello Matias,

Thanks for adding this issue!
However, I will upload the code on github once it is approved by 
Alessandro as we have already agreed upon it.

Thanks & Best regards
Khushbu

On 1/25/2015 10:06 PM, Matias Mora wrote:
> The TMCDB as developed for ALMA is supported on two databases, hsqldb, a
> lightweight java RDBMS suitable for unit testing or small installations,
> and Oracle, an expensive commercial DB chosen for ALMA production use.
> MySQL is an open-source alternative to Oracle, and in view of the
> unsuitability of Oracle (or probably any standard RDBMS) for the storage
> and retrieval of large time-series (monitor) data, probably sufficient
> for most configuration uses.
>
> The implementation has already been done by Khushbu Verma of DESY, and
> would need only to be reviewed and tested by the community, before
> integration as an alternative backend.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ACS-Community/ACS/issues/28>.
>

-- 
Khushbu Verma
DESY Zeuthen, Platanenallee 6, 15738 Zeuthen
Tel.:+49(33762)77456

**Comment by moraklein** [comment]
At the last ACB meeting Alessandro confirmed that this feature will be shipped by ESO with ACS-2015.2, as an alternative to hsqldb for testing. The CCB should decide if we wait for this release to be merged or integrate it right away to the community Trunk.

**Comment by acaproni** [comment]
Matias, is there a special reason let the CCB decide to integrate immediately instead of waiting for the next release? 
It sounds to me like you're proposing to do something that we have already implemented (duplication of work?)

**Comment by javarias** [comment]
Alessandro,

First of all, what feature ACS community decide to accept and merge is a decision that CCB takes, I think that is not you call.
Second, We think that merging this feature is really good exercise to define how we will accept features into the main ACS community repository. It is true that the standards are not completely defined yet, but going through this procedure will help to clarify how and what we need to accept.

**Comment by jhschwarz** [comment]
As a member of the CCB I think that Ale is entitled to his opinion, and what's more, I agree with him. Let's spend our few resources on things that ESO doesn't provide.

**Comment by moraklein** [comment]
OK, I think we can all agree that there will be no duplication of work here. Since Alessandro already integrated this feature into the ESO release, we're thankful for his help and the CCB will pick it up when merging ACS-2015.2 into the community Trunk.
Lets try to make an effort on all sides to better communicate ongoing work in the future.

**Comment by acaproni** [comment]
I hope that everybody in this community can freely express its own 
opinion from the time being because replies like the one by Jorge will 
not help to build up any good relationship. I agree with Matias that the 
community needs better communication but we can't accept lack of respect 
and politeness.

I am also wondering why the decision to close this ticket has taken by 
Matias that is not part of the CCB as I am not.
